Bitkit/Views/Transfer/SpendingAdvancedView.swift

```diff
@@ -99,10 +102,15 @@ struct SpendingAdvancedView: View {
                 blocktankInfo: blocktank.info
             )
 
+            amountViewModel.updateFromSats(transfer.transferValues.defaultLspBalance, currency: currency)
```

Comment:
Initial balance should be 0 on this screen

Reply:
IMO if the user navigates to this screen they intend to set a custom amount, defaulting to the amount that was already set is forcing them to delete via keyboard first. I would prefer if we don't make UX changes without discussion with design team first, even small ones.